src/app/modules/practitioner-registration/pages/practitioner-attachment/practitioner-attachment.component.ts

```diff
@@ -78,12 +81,16 @@ export class PractitionerAttachmentComponent extends BcpBaseForm implements OnIn
     }
     if (this.dataService.attachmentEffectiveDate
       && this.dataService.attachmentCancelDate
-      && isBefore(this.dataService.attachmentCancelDate, this.dataService.attachmentEffectiveDate)) {
+      && (isBefore(this.dataService.attachmentCancelDate, this.dataService.attachmentEffectiveDate)
+        || isSameDay(this.dataService.attachmentEffectiveDate, this.dataService.attachmentCancelDate)
```

Comment:
if ( this.dataService.attachmentCancelDate ) {
return isAfter( this.dataService.atttachmentCancelDate, this.dataService.facCancelDate) ? this.dataService.facCancelDate : this.dataService.atttachmentCancelDate:
}
return this.dataService.facCancelDate;